controllers/flowlogspipeline/flp_monolith_reconciler.go

```diff
@@ -40,6 +40,7 @@ func newMonolithReconciler(info *reconcilersCommonInfo) *flpMonolithReconciler {
 		promService:    &corev1.Service{},
 		serviceAccount: &corev1.ServiceAccount{},
 		configMap:      &corev1.ConfigMap{},
+		dbConfigMap:    &corev1.ConfigMap{},
```

Comment:
this new config map should also be added with `AddManagedObject`, same as the existing configmap. This is necessary to manage namespace switching: if `netobserv` is moved to a different namespace, it ensures the previous configmap is correctly going to be deleted.

Reply:
(same applies to `flp_transfo_reconciler` file)

Reply:
Yes, that is what I thought at first. But then I ran into an issue that this new config map is not really in the `netobserv` namespace. It technically is in the `openshift-config-managed` namespace, but we have to set it up when we create all the objects in the `netobserv` namespace. When the `netobserv` namespace is moved to a different name, it should not affect the dashboard objects.

Reply:
Oh I didn't notice that, thanks for the explaination